git_wrapper/base.py

```diff
@@ -75,7 +79,7 @@ def describe(self, sha):
         try:
             output = self.git.describe('--all', sha).split('-g')
         except git.CommandError as ex:
-            raise_from(exceptions.DescribeException("Error while running describe command on sha %s: %s" % (sha, ex)), ex)
+            raise_from(exceptions.DescribeException("Error while running describe command on sha {sha}: {error}".format(sha=sha, error=ex)), ex)
```

Comment:
This line is extremely long, I can't even easily read it in the PR, can you wrap it when you get a chance?

Reply:
Done.